samples/apps/bookinfo/bookinfo.yaml

```diff
@@ -156,7 +184,57 @@ spec:
         - containerPort: 9080
 ---
 ##################################################################################################
-# Productpage service
+# Mysql db services
+# credentials: root/password
+##################################################################################################
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mysql-credentials
+type: Opaque
+data:
+  rootpasswd: cGFzc3dvcmQ=
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: mysqldb
+  labels:
+    app: mysqldb
+spec:
+  ports:
+  - port: 3306
+    name: mysql
+  selector:
+    app: mysqldb
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
```

Comment:
Would it be better to launch mysql outside of the mesh? A more realistic scenario? I wonder if it even works, since several users have been noticing issues with accessing non-istiofied TCP and/or headless services. Would be a good test anyway.